### PR TITLE
Adds support of bucket sync disable and enable feature

### PR DESCRIPTION
**Pull request by udaysk23**
Adds support of bucket sync disable and enable feature
Polarion id: https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83574434
Log link: http://pastebin.test.redhat.com/1093769

### CODE REVIEW

File: rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py [hunk 1]
```diff
@@ -529,6 +543,30 @@ def test_exec(config, ssh_con):
...
+                                reusable.upload_object(
+                                    s3_object_name,
+                                    bucket,
+                                    TEST_DATA_PATH,
+                                    config,
+                                    each_user,
+                                )
+                            op = utils.exec_shell_cmd(
+                                f"radosgw-admin bucket sync enable --bucket {bkt}"
+                            )
```
Comment (mkasturi18) [inline rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py:568]: @udaysk23 could we call the function check_bucket_sync_status here and validate the status

File: rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py [hunk 2]
```diff
@@ -505,6 +505,20 @@ def test_exec(config, ssh_con):
...
                                 )
                     if config.manual_resharding is True:
+                        if config.sync_disable_and_enable:
+                            reusable.check_sync_status()
+                            op = utils.exec_shell_cmd(
+                                f"radosgw-admin bucket sync disable --bucket {bkt}"
+                            )
+                            bucket_sync_status = reusable.check_bucket_sync_status(bkt)
+                            for i in range(10):
+                                if "disabled" in bucket_sync_status:
```
Comment (mkasturi18) [inline rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py:515]: @udaysk23 , could you add a log.info

Comment (udaysk23) [reply rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py:515]: Done

**Comment by udaysk23** [comment]
Pass log with new code: http://pastebin.test.redhat.com/1093986

**Review by mkasturi18 — APPROVED** [review]
Requested changes are in.

**Pull request merged**